torchaudio/models/_utils.py

```diff
@@ -0,0 +1,8 @@
+try:
```

Comment:
Since this `_utils` is only used by WaveRNN and we do not have a plan to use this in other place, can you place this in `wavernn.py` directly?

Reply:
What I was originally thinking is that if we want to add pertained weights for [Tacotron2](https://github.com/pytorch/audio/pull/1621) in the future, we may want to use the similar function. But since this is only an internal function, the cost of moving this function back to _utils.py should be low in the future. So I am okey with either way. Do you still think it is better to keep this function in `wavernn.py`?

Reply:
I've removed this file for now. We can decide whether to put it back in the future if needed.